src/main/java/io/deepstream/GlobalConnectivityState.java

```diff
@@ -0,0 +1,16 @@
+package io.deepstream;
+
+/**
```

Comment:
can we remove this comment and replace with a small description of what the enums are used for